_topic_maps/_topic_map.yml

```diff
@@ -174,6 +174,8 @@ Topics:
     File: installing-aws-china
   - Name: Installing a cluster on AWS using CloudFormation templates
     File: installing-aws-user-infra
+  - Name: Installing a cluster on AWS extending workers to the Local Zones
```

Comment:
I'd replace every entry referring to "Local Zones" with "AWS Local Zones". Some parts of the documentation are using the full "AWS Local Zones" name already, for consistency I'd keep it this way for any reference to Local Zones

Reply:
Changed to "Installing a cluster using AWS Local Zones". Thank You!